charts/retool/templates/_helpers.tpl

```diff
@@ -23,6 +23,30 @@ If release name contains chart name it will be used as a full name.
 {{- end }}
 {{- end }}
 
+{{/*
+Set service names
+*/}}
+{{- define "retool.codeExecutor.name" -}}
+  {{- default (printf "%s-code-executor" (include "retool.fullname" .)) .Values.codeExecutor.name -}}
+{{- end -}}
+
+{{- define "retool.backend.name" -}}
+  {{- default (include "retool.fullname" .) .Values.backend.name -}}
+{{- end -}}
+
+{{- define "retool.jobsRunner.name" -}}
+  {{- default (printf "%s-jobs-runner" (include "retool.fullname" .)) .Values.jobRunner.name -}}
+{{- end -}}
+
+{{- define "retool.workflowsWorker.name" -}}
+  {{- default (printf "%s-workflow-worker" (include "retool.fullname" .)) .Values.workflows.worker.name -}}
+{{- end -}}
+
+{{- define "retool.workflowsBackend.name" -}}
+  {{- default (printf "%s-workflow-backend" (include "retool.fullname" .)) .Values.workflows.backend.name -}}
+{{- end -}}
+
+
```

Comment:
enable internal Retool to align service names with Cloud Prod, for DD dashboards shared across env